`tools/team-generation-benchmark.js` fails because of `require` path

Running `node ./tools/team-generation-benchmark.js random` gives me this error
```
$ node ./tools/team-generation-benchmark.js random
node:internal/modules/cjs/loader:1143
  throw err;
  ^

Error: Cannot find module '../sim'
Require stack:
- /redacted/tools/team-generation-benchmark.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:1140:15)
    at Module._load (node:internal/modules/cjs/loader:981:27)
    at Module.require (node:internal/modules/cjs/loader:1231:19)
    at require (node:internal/modules/helpers:177:18)
    at Object.<anonymous> (/redacted/tools/team-generation-benchmark.js:3:17)
    at Module._compile (node:internal/modules/cjs/loader:1364:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
    at Module.load (node:internal/modules/cjs/loader:1203:32)
    at Module._load (node:internal/modules/cjs/loader:1019:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/redacted/tools/team-generation-benchmark.js'
  ]
}

Node.js v18.20.4
```

I think these lines got left behind during a previous build system migration.
https://github.com/smogon/pokemon-showdown/blob/269e6f3751289e5526f6a3a09dfc652c55a7d217/tools/team-generation-benchmark.js#L3-L4

It works when I add `dist/` to the path, but that doesn't seem to trigger a rebuild if the TS files change. I'm pretty clueless when it comes to compiling JS/TS so I don't know what the best fix is.

--- a/tools/team-generation-benchmark.js
+++ b/tools/team-generation-benchmark.js
@@ -1,7 +1,7 @@
 'use strict';
 
-const {Teams} = require('../sim');
-const {Dex} = require('../sim/dex');
+const {Teams} = require('../dist/sim');
+const {Dex} = require('../dist/sim/dex');
 
 
 if (!process.argv[2]) {

